mail/api_test.py

```diff
@@ -0,0 +1,55 @@
+"""
+Test cases for email API
+"""
+from unittest.mock import patch
+from django.test import TestCase
+
+from micromasters import settings
+from mail.api import MailgunClient
+
+
+@patch('requests.post')  # pylint: disable=missing-docstring
+class MailAPITests(TestCase):
+    def test_from_address(self, mock_post):  # pylint: disable=no-self-use
+        """
+        Test that the 'from' address for our emails is set correctly
+        """
+        # NOTE: Using patch.multiple to override settings values because Django's
+        # override_settings decorator fails to work for mysterious reasons
+        with patch.multiple('mail.api', MAILGUN_FROM_EMAIL='mailgun_from_email@example.com'):
```

Comment:
read the comment directly above. it simply isn't working. if you want to try it locally, please do. i dumped at least an hour trying to get that decorator to work